pdr_backend/ppss/test/test_sim_ss.py

```diff
@@ -61,16 +62,16 @@ def test_sim_ss_log_dir_relative_path():
 
 @enforce_types
 def test_sim_ss_test_n_badpaths(tmpdir):
-    d = sim_ss_test_dict(_logdir(tmpdir), test_n=-3)
+    d = sim_ss_test_dict(_logdir(tmpdir), True, test_n=-3)
     with pytest.raises(ValueError):
         _ = SimSS(d)
 
-    d = sim_ss_test_dict(_logdir(tmpdir))
+    d = sim_ss_test_dict(_logdir(tmpdir), True)
```

Comment:
can you use keyed arguments here? Otherwise it is ambiguous (at a glance) what the boolean refers to, without checking out the function itself.